<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Management System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">color:rgb(169, 182, 255)</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="loginLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameInput">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordInput">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isAdminCheckbox">
          <property name="font">
           <font>
            <family>Cabin</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Admin</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="signupButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Sign Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveUserButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Save User</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="restoreUserButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Restore User</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adminPage">
       <layout class="QVBoxLayout" name="adminLayout">
        <item>
         <widget class="QPushButton" name="addBookButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
           </font>
          </property>
          <property name="text">
           <string>Add Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButtonAdmin">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
           </font>
          </property>
          <property name="text">
           <string>Search Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterButtonAdmin">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
           </font>
          </property>
          <property name="text">
           <string>Filter Books</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeBookButton">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
           </font>
          </property>
          <property name="text">
           <string>Remove Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="databaseView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="userPage">
       <layout class="QVBoxLayout" name="userLayout">
        <item>
         <widget class="QPushButton" name="searchButtonUser">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Search Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterButtonUser">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Filter Books</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
